app/scripts/views/force_auth.js

```diff
@@ -23,20 +23,20 @@ define(function (require, exports, module) {
     return '';
   }
 
+  var proto = SignInView.prototype;
 
   var View = SignInView.extend({
     template: Template,
-    className: 'sign-in',
+    className: 'force-auth',
 
     initialize: function (options) {
-      options = options || {};
+      proto.initialize.call(this, options);
 
-      this._formPrefill = options.formPrefill;
-
-      // forceAuth means a user must sign in as a specific user.
+      // forceAuth means a user msut sign in as a specific user.
       // kill the user's local session.
+      // TODO - I think this should go too, but am unsure of
+      // the repercussions.
       Session.clear();
```

Comment:
@shane-tomlinson 

> Why do we do Session.clear here, is that old legacy code because all the user's session info used to be stored in Session?

Yeah looks like it, dates back to "first commits", kill it :+1: